### DISCUSSION TITLE
NGinx Reverse Proxy issue.

### DISCUSSION THREAD

**Discussion by Funasitien** [plan-player-analytics/Plan#3294] [Q&A] [answered]
So I have the same problem as #2554, but i'm using debian on Proxmox, with nginx and pterodactyyl on the same server.

If I use the ip adress 127.0.0.1, I got the same error as the issue above

BUT with any other ip adress (like localhost), I got the error `no live upstreams while connecting to upstream`

It said on forums to stay with the ip 127.0.0.1, but it dosen't work too...

Here is my PLAN config : https://pastebin.com/JcSa0rdv
NGinx Config : https://pastebin.com/SSLz8eAa

**Comment by AuroraLS3** [comment, marked as answer]
The proxy pass address needs to be the same as your port allocation in pterodactyl. After any allocation changes restart the server.

The wiki includes pterodactyl instructions https://github.com/plan-player-analytics/Plan/wiki/External-Webserver-use:-Reverse-proxy#external-webserver-use-reverse-proxy
